nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -116,7 +116,9 @@ public Nullness onOverrideMethodInvocationReturnNullability(
       Nullness returnNullness) {
     OptimizedLibraryModels optLibraryModels = getOptLibraryModels(state.context);
     if (optLibraryModels.hasNonNullReturn(methodSymbol, state.getTypes(), !isAnnotated)) {
-      return Nullness.NONNULL;
+      return NONNULL;
+    } else if (optLibraryModels.hasNullableReturn(methodSymbol, state.getTypes(), !isAnnotated)) {
+      return NULLABLE;
```

Comment:
For consistency. Without this change library models specifying an annotated method is `@Nullable` will be ignored while analyzing the return statements inside such methods. In theory this also changes behavior for overriding, but the error will manifest only in the case where we have a `@Nullable` model for a method which overriding either a) a method in annotated code or, b) a third-party method with a `@NonNull` return as a library model. Guess this is probably worth some internal testing just in case I am missing some subtlety about our existing library models...